omniscidb/QueryEngine/CostModel/IterativeCostModel.cpp

```diff
@@ -0,0 +1,40 @@
+#include "IterativeCostModel.h"
+#include "Dispatchers/ProportionBasedExecutionPolicy.h"
+
+#include <cmath>
+
+namespace costmodel {
+
+std::unique_ptr<policy::ExecutionPolicy> IterativeCostModel::predict(
+    QueryInfo queryInfo) {
+        std::lock_guard<std::mutex> l(latch_);
+
+    unsigned cpuProp, gpuProp;
+    size_t optStep = std::ceil(static_cast<float>(queryInfo.bytesSize) / optimizationIterations);
```

Comment:
Please stick to the style guide: https://google.github.io/styleguide/cppguide.html#Variable_Names